cookbook/readme.md

```diff
@@ -0,0 +1,6 @@
+# This is cookbook
+
+The cookbook is a place where we put recipes for things around MrChecker framework.
+
+* [small project template](./small_project_template) - set of basic CI/CD tools to conduct a small project
+* [huge project template](./huge_project_template) - set of best practices with examples for conducting huge projects
```

Comment:
Can we add model-based-testing as well? Or does model-based-testing need working on?

Reply:
I believe model base testing is already merged to Develop so I'm pretty suprised the merge conflict is not raised.

Reply:
NVM model based testing does not have this table of content page. Thats why there is no conflict